src/Spout/Writer/Style/Style.php

```diff
@@ -269,16 +269,25 @@ public function shouldWrapText()
     }
 
     /**
+     * @param boolean $shouldWrap Should the text be wrapped
```

Comment:
`@param bool|void` (for consistency, as we've been using "bool" instead of "boolean" everywhere and this is what PHP uses too)